pkg/apis/helm.fluxcd.io/v1/types.go

```diff
@@ -208,7 +226,7 @@ func (hr HelmRelease) GetValuesFromSources() []ValuesFromSource {
 }
 
 type HelmReleaseStatus struct {
-	// ReleaseName is the name as either supplied or generated.
+	// GetReleaseName is the name as either supplied or generated.
```

Comment:
I think this might have been an accidental change

Reply:
It was not accidental but changed to get the `HelmRelease` methods in a more uniform shape, as most values are retrieved through `Get<Value>` methods.